e2e/project/frontity.settings.ts

```diff
@@ -35,11 +35,7 @@ const settings: Settings = [
   },
   {
     name: "wp-source-errors",
-    packages: [
-      "e2e-wp-source-errors",
-      "@frontity/wp-source",
-      "@frontity/tiny-router"
-    ]
+    packages: ["e2e-wp-source-errors", "@frontity/wp-source"]
```

Comment:
@luisherranz I'm a bit confused. Why did you remove the `tiny-router`? 

I've added it because I needed to test that the `beforeSSR` *in the* tiny-router package adds the correct error message.

Reply:
I thought that it was not needed to test that we can change the `ctx.status` from a `beforeSSR` method... but it's fine. Add it back if you want.